c-dependencies/js-compute-runtime/Makefile

```diff
@@ -53,8 +58,33 @@ $(RUST_URL_LIB): $(RUST_URL_RS_FILES) $(RUST_URL_SRC)/Cargo.toml $(RUST_URL_SRC)
 	$(WASI_CXX) $(CXX_FLAGS) $(CXX_OPT) $(DEFINES) -I $(SM_SRC)include -MMD -MP -c -o $@ $<
 
 js-compute-runtime.wasm: $(FSM_OBJ) $(SM_OBJ) $(RUST_URL_LIB)
-	$(WASI_CXX) $(CXX_FLAGS) $(CXX_OPT) $(DEFINES) $(LD_FLAGS) -o $@ $^
-	$(WASM_STRIP)
+	$(WASI_CXX) $(CXX_FLAGS) $(CXX_OPT) $(DEFINES) $(LD_FLAGS) -o $@ $^ && \
+		$(call WASM_STRIP,$@)
```

Comment:
I haven't touched makefiles in a while, but the [GNU make manual](https://www.gnu.org/software/make/manual/html_node/Errors.html) says execution within a rule stops at the first line that exits with a non-zero status. So combining two lines with `&&` is unnecessary and I think makes maintenance a little harder.

Reply:
This avoids the overhead of invoking the shell once per line, but that's probably not really something worth worrying about here. I'll switch it back because I agree that it's more readable on a separate line :+1: 

There is the `.ONESHELL` directive, but that brings back the opportunity for a command to fail and not halt rule execution.